charts/agent/values.schema.json

```
@@ -23,6 +24,100 @@
           ]
         }
       }
+    },
+    "localForwarder": {
+      "type": "object",
+      "required": [
+        "enabled",
+        "integrations",
+        "transmitMessageTypes"
+      ],
+      "if": {
+        "properties": {
+          "enabled": {
+            "const": true
+          }
+        }
+      },
+      "then": {
+        "properties": {
+          "integrations": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "$ref": "#/$defs/integration"
+            }
+          },
+          "enabled": {
+            "type": "boolean"
+          },
+          "transmitMessageTypes": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "AUDIT_TAP",
+                "POLICY_EVENTS",
+                "SECURE_AUDIT"
+              ]
+            }
+          }
+        }
+      }
+    }
+  },
+  "$defs": {
+    "integration": {
```

let's use a less ambiguous name here ;) 
```suggestion
    "AgentLocalForwarderIntegration": {
```

---

Handled in 372a28b